src/utils/Debounce.ts

```diff
@@ -1,8 +1,8 @@
 export const debounce = (func: any, wait: number, other: any, x: any, y: any) => {
   const cleanTimeout = setTimeout(() => {
+    other;
```

Comment:
??

Reply:
other is the clear recipe list function, i'll rename it to make it clearer